CoreImportExportPlugins/src/au/gov/asd/tac/constellation/plugins/importexport/jdbc/JDBCSourcePane.java

```diff
@@ -304,11 +305,15 @@ public JDBCSourcePane(final JDBCImportController importController) {
         final Button queryButton = new Button("Query");
         queryButton.setOnAction((final ActionEvent t) -> {
             if (!query.getText().isBlank() && dbConnectionComboBox.getValue() != null) {
+
                 importController.setDBConnection(dbConnectionComboBox.getValue());
                 importController.setQuery(query.getText());
                 importController.setUsername(username.getText());
                 importController.setPassword(password.getText());
                 importController.updateSampleData();
+            } else {
+                NotifyDisplayer.displayLargeAlert("JDBC Import", "No Query Entered",
```

Comment:
Are you able to possibly change this from `displayLargeAlert` to `displayAlert`? Given the size of the message, I think the latter should be ok.